samples/can/src/main.c

```diff
@@ -0,0 +1,294 @@
+/*
+ * Copyright (c) 2018 Alexander Wachter
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <kernel.h>
+#include <misc/printk.h>
+#include <device.h>
+#include <can.h>
+#include <gpio.h>
+
+#define CAN_DEV "CAN_1"
+#define GPIO_LED_DEV "GPIOC"
```

Comment:
You could place all these board specific options to sample specific Kconfig file.

Reply:
do you rally think this is a good idea?
In my opinion samples are not configured by Kconfig.
Can you give me an example which sample code uses this?

Reply:
See for example https://github.com/zephyrproject-rtos/zephyr/tree/master/samples/net/stats
My point was to put board specific stuff in sample app to Kconfig so that user does not have to change the actual code if he wants to run in different board where the settings might be different.
You need to also add `set(KCONFIG_ROOT ${CMAKE_CURRENT_SOURCE_DIR}/Kconfig)` to CMakeLists.txt for this to work.

Reply:
Sounds reasonable. I'll do that.